sdk/test/opentelemetry/sdk/trace/tracer_test.rb

```diff
@@ -329,5 +329,28 @@
       _(span.status.code).must_equal(OpenTelemetry::Trace::Status::ERROR)
       _(span.status.description).must_equal('Unhandled exception of type: RuntimeError')
     end
+
+    it 'yields a no-op span within an untraced block' do
+      tracer_provider.sampler = Samplers::ALWAYS_ON
```

Comment:
Do we really want untraced to bypass ALWAYS_ON? 

Is they what other Language SDKs do?

Reply:
oh, this line is not needed for this PR, it seems I copy-pasted too much. I'm going to remove it.

> Do we really want untraced to bypass ALWAYS_ON?

I think this is a separate topic and unrelated to this PR...